pkg/wkspace/minimal.go

```diff
@@ -56,107 +69,131 @@ func FormatValues(w io.Writer, vals string, output *output.Output) (err error) {
 	return
 }
 
-func templateVals(app, path string) (backup string, err error) {
-	root, _ := utils.ProjectRoot()
-	valsFile := pathing.SanitizeFilepath(filepath.Join(path, defaultValuesYaml))
-	vals, err := utils.ReadFile(valsFile)
+func (m *MinimalWorkspace) BounceHelm(extraArgs ...string) error {
+	path, err := filepath.Abs(pathing.SanitizeFilepath(filepath.Join("helm", m.Name)))
 	if err != nil {
-		return
+		return err
 	}
-
-	out, err := output.Read(pathing.SanitizeFilepath(filepath.Join(root, app, "output.yaml")))
+	defaultVals, err := getValues(m.Name)
 	if err != nil {
-		out = output.New()
+		return err
+	}
+	namespace := m.Config.Namespace(m.Name)
+	actionConfig := new(action.Configuration)
```

Comment:
could we refactor this helm cli setup into a helper function?  I feel like it benefits from being reusable